tests/test_agents.py

```diff
@@ -352,12 +352,12 @@ async def on_agent_action(  # noqa: RUF029
     async def llm_model_call(*args, **kwargs):
         # NOTE: "required" will not lead to thoughts being emitted, it has to be "auto"
         # https://docs.anthropic.com/en/docs/build-with-claude/tool-use#chain-of-thought
-        kwargs.pop("tool_choice", MultipleCompletionLLMModel.TOOL_CHOICE_REQUIRED)
+        # kwargs.pop("tool_choice", LiteLLMModel.TOOL_CHOICE_REQUIRED)
+        # tool_choice is now a arg, not a kwarg
+        args = args[:-1]  # removing last element from args
```

Comment:
```suggestion
        args = args[:-1]  # removing last element (tool_choice) from args
```

No need for legacy comments